app/helpers/export_helper.rb

```diff
@@ -223,8 +223,8 @@ def write_work_exports(works, out, export_user, bulk_export)
 
 
   def work_to_xhtml(work)
-    @work = Work.includes(pages: [{notes: :user}, {page_versions: :user}]).find_by(id: work.id)
-    render_to_string :layout => false, :template => "export/show.html.erb"
+    @work = Work.includes(pages: [{ notes: :user }, { page_versions: :user }]).find_by(id: work.id)
+    render_to_string template: 'export/show', layout: false, formats: [:html], handlers: [:erb]
```

Comment:
This syntax seems to be not supported in newer versions of rails (I got multiple failures due to doing relative path with extention, ex: `deed/deed.html`, `export/show.html.erb`).


We instead pass in `formats` and `handlers` params